scipy/stats/_boost/include/code_gen.py

```diff
@@ -0,0 +1,154 @@
+'''First stab at generating PYX files.'''
```

Comment:
I'm not sure when or how, but this file deserves a little more work.  It was written as a quick prototype and I would have done things differently if I had known it was going in a PR.  Adding this comment to remind myself.